### PR TITLE
Popup component

### PR DESCRIPTION
**Pull request by med-hedi**

### CODE REVIEW

File: src/components/contingency-list.js [hunk 1]
```diff
@@ -313,57 +228,51 @@ const Contingency = () => {
...
-                    setDisabledBtnRenameList(true);
-                }
-            } else {
-                setDisabledBtnRenameList(true);
-            }
-        }
+    const addNewList = (name) => {
+        aceEditorRef.current.editor.setValue('');
+        setAlertEmptyList(false);
+        setSelectedIndex('');
```
Comment (ChamseddineBhd) [inline src/components/contingency-list.js]: Index should be a number

File: src/components/contingency-list.js [hunk 2]
```diff
@@ -592,7 +500,7 @@ const Contingency = () => {
                                     </>
                                 ))}
                             </List>
-                            {/* To be replaced by snackbar */}
+                            {/* To be replaced width snackbar */}
```
Comment (ChamseddineBhd) [inline src/components/contingency-list.js]: with*

File: src/components/popup.js [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+    const handleClose = () => {
+        onClose(false);
+    };
+
+    return (
+        <DialogContainer open={open} onClose={handleClose}>
+            <CustomDialogTitle onClose={handleClose}>{title}</CustomDialogTitle>
+            <CustomDialogContent dividers>
+                <TextField
+                    defaultValue={!newList ? selectedListName : ''}
```
Comment (ChamseddineBhd) [inline src/components/popup.js]: !newList ? selectedListName : ''  
-> newList ? " " :  selectedListName

**Review by ChamseddineBhd — APPROVED** [review]

**Comment by med-hedi** [comment]
Following the merge of the gui-mode branch in master, we no longer need to merge this branch because the code of this branch was used in gui-mode

**Pull request closed without merging**